contracts/system_tests/TestNFTRewardsFlow.sol

```diff
@@ -0,0 +1,166 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.6;
+
+import './helpers/console.sol';
```

Comment:
should we keep this (+we used emit log until now, but up's to anyone's choice:)?

Reply:
that would be cool, its useful for amateurs like me

Reply:
> that would be cool, its useful for amateurs like me

yh yh, I use it all the time too, but `emit log(..)` is part of ds-test already (no need to add a new helper then)